### DISCUSSION TITLE
Ainda existem desenvolvedores que usam PageBuilders?

### DISCUSSION THREAD

**Discussion by marcusvcribeiro** [frontendbr/forum#975] [Perguntas e Respostas]
Hoje em uma discussão em grupo de whatsapp, vi alguns "Desenvolvedores" usando o Adobe Muse, um page builder  :-1:   E defendendo pesado sobre o uso.

Ai fiquei pensando, ainda existem desenvolvedores que usam pageBUilders?

**Comment by kivervinicius** [comment]
Na teoria se existe a ferramenta há público que use! Existem hoje inúmeros plugins para wordpress que fazem isso, já até vi quem utiliza até hoje Dreamweaver para gerar CRUD'S PHP, agora se é viável, depende, pra um site que o cara quer cobrar 100 R$ ta bom demais! rs

**Comment by tarciozemel** [comment]
Muita gente usa, incluindo "leigos" que não têm ideia do que seja um código-fonte, mas querem/precisam vender um site.

Pessoalmente, não vejo nada de errado nisso. Existem soluções diferentes para problemas/necessidades diferentes. Se alguém oferece um produto que facilita a coisa, tudo bem.  :-)

**Comment by Cleysonlb** [comment]
Toda solução resolve uma necessidade, se o site criado por um `pagebuilder` atende, não vejo problema algum em o cara usar e ganhar a graninha dele, se tem gente que paga é pq tem mercado, se tem mercado tem que existir quem faça.

**Comment by felipefialho** [comment]
Essas soluções atingem um público especifico. Não vejo problema nelas. Se resolvem os problemas que visam resolver, são úteis.
